### PR TITLE
Add label & message color properties to VisualStyle

### PR DESCRIPTION
**Pull request by virusman**
Simple change to allow customization of alert colors.

### CODE REVIEW

File: SDCAlertView/Source/SDCAlertControllerDefaultVisualStyle.m [hunk 1]
```diff
@@ -44,6 +44,14 @@ - (UIFont *)messageLabelFont {
...
 }
 
+-(UIColor *)titleLabelColor {
+	return [UIColor blackColor];
+}
+
+-(UIColor *)messageLabelColor {
+	return [UIColor blackColor];
+}
+
```
Comment (sberrevoets) [inline SDCAlertView/Source/SDCAlertControllerDefaultVisualStyle.m]: Nitpick: can you add a space between the `-` and `(UIColor *)` so it's consistent with the other methods?

**Comment by sberrevoets** [comment]
I'll merge this in after my nitpick, but in a future version (probably 3.0) I'm probably going to simplify this by changing it to attributes that can be applied to an attributed string.

Thanks for your contribution!

**Comment by virusman** [comment]
Done, added the spaces.

**Comment by sberrevoets** [comment]
And merged. Thanks again!

**Pull request merged**